easybuild/easyconfigs/o/orjson/orjson-3.9.15-GCCcore-12.3.0.eb

```
@@ -16,6 +16,620 @@ dependencies = [
+    {'ident_case-1.0.1.tar.gz': 'b9e0384b61958566e926dc50660321d12159025e767c18e043daf26b70104c39'},
+    {'idna-0.5.0.tar.gz': '634d9b1461af396cad843f47fdba5597a4f9e6ddd4bfb6ff5d85028c25cb12f6'},
+    {'ignore-0.4.22.tar.gz': 'b46810df39e66e925525d6e38ce1e7f6e1d208f72dc39757880fcb66e2c58af1'},
+    {'imperative-1.0.5.tar.gz': '8b70798296d538cdaa6d652941fcc795963f8b9878b9e300c9fab7a522bd2fc0'},
+    {'indexmap-2.2.6.tar.gz': '168fb715dda47215e360912c096649d23d58bf392ac62f73919e831745e40f26'},
+    {'inotify-0.9.6.tar.gz': 'f8069d3ec154eb856955c1c0fbffefbf5f3c40a104ec912d4797314c1801abff'},
+    {'inotify-sys-0.1.5.tar.gz': 'e05c02b5e89bff3b946cedeca278abc628fe811e604f027c45a8aa3cf793d0eb'},
+    {'insta-1.39.0.tar.gz': '810ae6042d48e2c9e9215043563a58a80b877bc863228a74cf10c49d4620a6f5'},
+    {'insta-cmd-0.6.0.tar.gz': 'ffeeefa927925cced49ccb01bf3e57c9d4cd132df21e576eb9415baeab2d3de6'},
+    {'is-docker-0.2.0.tar.gz': '928bae27f42bc99b60d9ac7334e3a21d10ad8f1835a4e12ec3ec0464765ed1b3'},
+    {'is-macro-0.3.5.tar.gz': '59a85abdc13717906baccb5a1e435556ce0df215f242892f721dff62bf25288f'},
+    {'is-wsl-0.4.0.tar.gz': '173609498df190136aa7dea1a91db051746d339e18476eed5ca40521f02d7aa5'},
+    {'itertools-0.13.0.tar.gz': '413ee7dfc52ee1a4949ceeb7dbc8a33f2d6c088194d9f922fb8318faf1f01186'},
+    {'itoa-1.0.11.tar.gz': '49f1f14873335454500d59611f1cf4a4b0f786f9ac11f4312a78e4cf2566695b'},
+    {'jod-thread-0.1.2.tar.gz': '8b23360e99b8717f20aaa4598f5a6541efbe30630039fbc7706cf954a87947ae'},
+    {'js-sys-0.3.69.tar.gz': '29c15563dc2726973df627357ce0c9ddddbea194836909d655df6a75d2cf296d'},
+    {'kqueue-1.0.8.tar.gz': '7447f1ca1b7b563588a205fe93dea8df60fd981423a768bc1c0ded35ed147d0c'},
+    {'kqueue-sys-1.0.4.tar.gz': 'ed9625ffda8729b85e45cf04090035ac368927b8cebc34898e7c120f52e4838b'},
+    {'lazy_static-1.4.0.tar.gz': 'e2abad23fbc42b3700f2f279844dc832adb2b2eb069b2df918f455c4e18cc646'},
+    {'libc-0.2.155.tar.gz': '97b3888a4aecf77e811145cadf6eef5901f4782c53886191b2f693f24761847c'},
+    {'libcst-1.4.0.tar.gz': '10293a04a48e8b0cb2cc825a93b83090e527bffd3c897a0255ad7bc96079e920'},
+    {'libcst_derive-1.4.0.tar.gz': 'a2ae40017ac09cd2c6a53504cb3c871c7f2b41466eac5bc66ba63f39073b467b'},
+    {'libmimalloc-sys-0.1.38.tar.gz': '0e7bb23d733dfcc8af652a78b7bf232f0e967710d044732185e561e47c0336b6'},
+    {'libredox-0.1.3.tar.gz': 'c0ff37bd590ca25063e35af745c343cb7a0271906fb7b37e4813e8f79f00268d'},
+    {'linked-hash-map-0.5.6.tar.gz': '0717cef1bc8b636c6e1c1bbdefc09e6322da8a9321966e8928ef80d20f7f770f'},
+    {'linux-raw-sys-0.4.13.tar.gz': '01cda141df6706de531b6c46c3a33ecca755538219bd484262fa09410c13539c'},
+    {'log-0.4.21.tar.gz': '90ed8c1e510134f979dbc4f070f87d4313098b704861a105fe34231c70a3901c'},
+    {'lsp-server-0.7.6.tar.gz': '248f65b78f6db5d8e1b1604b4098a28b43d21a8eb1deeca22b1c421b276c7095'},
+    {'lsp-types-0.95.1-3512a9f.tar.xz': 'c766c15dba6e709f5c6e1cb275bd9f49bdeadf2a7ff48dbe9112ce98374810dc'},
```

```suggestion
    {'lsp-types-0.95.1-3512a9f.tar.xz': '8b52ec5508fdad182d1df1ad008ab6abdbd197c8e4a1bcc5eea65d60f4213c99 '},
```

checksum issue is due to not generating reproducible checksums as I'm on Python 3.6